rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] Please modify the constructor of the `Rental` class and pass in `book` and `person` so that you can set it inside 👍

Reply:
@codepantha Thank you for your feedback.